dev-tools/reconfigurator-cli/src/lib.rs

```diff
@@ -371,6 +374,20 @@ impl From<SledPolicyOpt> for SledPolicy {
     }
 }
 
+#[derive(Debug, Args)]
+struct SledUpdateSpArgs {
+    /// id of the sled
```

Comment:
Nit
```suggestion
    /// ID of the sled
```

Reply:
Heh, interesting.  I was curious what we generally do here.  It looks like we pretty consistently use `id` in the help output of all the other commands here.  19 out of 20 of them do that.  I'll stick with `id` here.